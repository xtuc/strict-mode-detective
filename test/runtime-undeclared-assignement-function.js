function hi() {
  c = 1;
}
hi();
