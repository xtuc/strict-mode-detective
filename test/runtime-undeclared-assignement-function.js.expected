  1 | function hi() {
> 2 |   c = 1;
    |  ^^^^^
  3 | }
  4 | hi();
  5 | 
undeclared assignement 

